docs/source/release_notes.rst

```diff
@@ -2,6 +2,7 @@
 -------------
 **Future Releases**
     * Enhancements
+        * Limited estimator usage for long-running multiclass problems on ``DefaultAlgorithm`` :pr:`3099`
```

Comment:
This description could be misleading since we already had this behavior for `IterativeAlgorithm`, but this implies this is new behavior specific to `DefaultAlgorithm`.